src/components/dropdown.ts

```diff
@@ -1,16 +1,15 @@
-import {Component, View} from 'angular2/angular2';
+import {Component, Input} from 'angular2/angular2';
 
-@Component({selector: 'ngb-dropdown', properties: ['open']})
-@View({
+@Component({
+  selector: 'ngb-dropdown',
   template: `<div class="dropdown" [class.open]="open">
     <ng-content></ng-content>
   </div>`
 })
 export class NgbDropdown {
-  private open: boolean;
+  @Input() private open: boolean;
```

Comment:
Yeah. Took me a while until I saw where it is defined in the source.

https://github.com/angular/angular/blob/master/modules/angular2/src/core/metadata/directives.ts#L977-L1025